sbirez/management/commands/importtopics.py

```diff
@@ -0,0 +1,52 @@
+from .modules.copy_topic_csvs import load
+from django.core.management.base import BaseCommand
+from django.core.management import call_command, CommandError
+from django.conf import settings
+import subprocess
+import os
+
+class Command(BaseCommand):
+    help = """Populates Topics with an .mdb dump.
+
+    Usage: python manage.py importtopics <path to .mdb file> "<name of solicitation>"
+    (solicitation name e.g. "DoD SBIR 2015.1"; use quotation marks)
+    """
+
+    def add_arguments(self, parser):
+
+        parser.add_argument('mdb_filename')
+        parser.add_argument('solicitation_name')
+        # example: DoD SBIR 2015.1
+        parser.add_argument('--clear', action='store_true',
+                            default=False,
+                            help='Firts, delete all records with this solicitation')
+
+    def handle(self, mdb_filename, solicitation_name, clear=False, **options):
+
+        if not os.path.isfile(mdb_filename):
+            raise OSError("MS Access database %s not found" % mdb_filename)
+
+        # Confirm that solicitation_name is for solicitation in the database
+        proc = subprocess.Popen(#'ls',
+                                '/usr/bin/psql -t -c "SELECT name FROM sbirez_solicitation" %s' %
```

Comment:
Is there a unix-y way of getting the path for psql? I've got it installed in `/usr/local/bin/psql`.